src/golang/lib/collections/workflow_dag_result/workflow_dag_result.go

```diff
@@ -9,14 +9,18 @@ import (
 	"github.com/aqueducthq/aqueduct/lib/collections/user"
 	"github.com/aqueducthq/aqueduct/lib/collections/workflow"
 	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/dropbox/godropbox/errors"
 	"github.com/google/uuid"
 )
 
+var ErrInvalidPendingTimestamp = errors.New("Execution state doesn't have a valid pending_at timestamp.")
+
 type WorkflowDagResult struct {
-	Id            uuid.UUID              `db:"id" json:"id"`
-	WorkflowDagId uuid.UUID              `db:"workflow_dag_id" json:"workflow_dag_id"`
-	Status        shared.ExecutionStatus `db:"status" json:"status"`
-	CreatedAt     time.Time              `db:"created_at" json:"created_at"`
+	Id            uuid.UUID                 `db:"id" json:"id"`
+	WorkflowDagId uuid.UUID                 `db:"workflow_dag_id" json:"workflow_dag_id"`
+	Status        shared.ExecutionStatus    `db:"status" json:"status"`
+	CreatedAt     time.Time                 `db:"created_at" json:"created_at"`
```

Comment:
Are you planning to deprecate `CreatedAt`?

Reply:
Yes, there are too many APIs (mainly in frontend) using that field so we would need to do that after we clean up these APIs

Reply:
https://linear.app/aqueducthq/issue/ENG-1701/deprecate-createdat-field-for-workflow-dag-results